Allan690/StoreManager-V2
# #161572742 Create README file and configure travis

#### What does this PR do?
Creates a README file and configures Travis for the application repo
#### Description of Task to be completed?
1. Create a README file
2. Add badges to the file
#### How should this be manually tested?
1. Navigate to the main page of [this](url) repo and click on the develop branch.
An updated README should be displayed.
#### Any background context you want to provide?
None
#### What are the relevant pivotal tracker stories?
[#161572742](https://www.pivotaltracker.com/story/show/161572742)
